website/content/docs/commands/acl/bootstrap.mdx

```diff
@@ -25,10 +28,11 @@ The `acl bootstrap` command requires no arguments.
 
 - `-json` : Output the bootstrap response in JSON format.
 - `-t` : Format and display the deployments using a Go template.
+- `-bootstrap-token` : Provide an operator generated management token.
```

Comment:
In addition to this document, we should add a line about the bootstrap token header to the [HTTP API docs for ACL tokens](https://github.com/hashicorp/nomad/blob/main/website/content/api-docs/acl-tokens.mdx)